shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/context/ApplicationContextTest.groovy

```diff
@@ -210,17 +210,14 @@ class ApplicationContextTest extends Specification {
         def injectionInstantiation = new InjectionDeclaringClass()
 
         TestApplicationContext.register(List.class, injectionInstantiation)
-        // notice above that I'm registering the injectionInstantiation object as a List class.
-        // injectionInstantiation is of class InjectionDeclaringClass,
-        // and InjectionDeclaringClass doesn't implement List (it only implements AFieldInterface).
```

Comment:
Do we no longer want these comments to point out the (on purpose) inconsistency?

Reply:
I'll add it back so it can provide the description